accepted/00000-salt-ssh-with-salt-bundle.md

```diff
@@ -0,0 +1,77 @@
+- Feature Name: Salt-SSH with Salt Bundle
+- Start Date: 2021-12-08
+
+# Summary
+[summary]: #summary
+
+Use Salt Bundle to handle Salt-SSH sessions on managed system side.
```

Comment:
```suggestion
Use Salt Bundle to handle Salt-SSH sessions on the minion side.
```

Unless there is a reason to, I suggest to stick to "minion" everywhere in this document.